.sequelizerc

```diff
@@ -0,0 +1,7 @@
+const path  = require('path')
+module.exports = {
+    "config": path.resolve('./src/database/config', 'config.js'),
+    "models-path": path.resolve('./src/database/models'),
+    "seeders-path": path.resolve('./src/database/seeders'),
+    "migrations-path": path.resolve('./src/database/migrations')
+};
```

Comment:
You can always add a `new line` at the end of a file, one of `eslint` rules. Good work

Reply:
@Naimaoye please add a new line here. Thanks